pkgs/games/openra/engine.nix

```diff
@@ -0,0 +1,39 @@
+# Based on: https://build.opensuse.org/package/show/home:fusion809/openra-ura
+{ stdenv
+, packageAttrs
+, patchEngine
+, wrapLaunchGame
+, engine
+}:
+
+with stdenv.lib;
+
+stdenv.mkDerivation (recursiveUpdate packageAttrs {
+  name = "${engine.name}-${engine.version}";
```

Comment:
```
pname = engine.name;
version = engine.version;
```

Reply:
I have gotten different feedback in the past, that it is better to keep those in their own let, instead of making them part of the derivation attributes, and if so then you do not gain anything (putting `version = engine.version` in a let, that is). So this is a bit of contradictory feedback to me, could you give any good arguments as to why I should ignore my previous feedback over yours?

Reply:
Correct, in the past it was undesirable because these weren't parameters to `stdenv.mkDerivation`, however, `pname` and `version` are nowadays parameters of `stdenv.mkDerivation`.

Reply:
Adding `name` should not hurt, so I have just added it explicitly as well.